Utility/Framework/SymfonyFramework.php

```diff
@@ -18,24 +18,6 @@ class SymfonyFramework
     /**
      * @return bool
      */
-    public static function hasDefinitionSharing()
```

Comment:
Definitely agreed about the 2.x API being unstable so it's ok to break atm until we are settled, we'd also be doing this by killing off the GridFS driver. 🙂 
Happy to brush this one up then!